api/iodeapi.h

```diff
@@ -62,7 +62,16 @@ extern int IodeSetVector(char *la_name, double *la_values, int la_pos, int ws_po
 extern int IodeEstimate(char* veqs, char* afrom, char* ato);
 
 extern int IodeModelSimulate(char *per_from, char *per_to, char *eqs_list, char *endo_exo_list, double eps, double relax, int maxit, int init_values, int sort_algo, int nb_passes, int debug, double newton_eps, int newton_maxit, int newton_debug);
-
+extern int IodeModelCalcSCC(int nbtris, char* pre_listname, char* inter_listname, char* post_listname, char *eqs_list);
```

Comment:
For your information, these IODE(...) already exist in some manner in the C++ API. 
Those C++ methods are actually called by the Qt GUI.

It's a bit of a shame to redo what's already done, but so be it.
I understand that you need these Iode(...) functions and to use Cython to have both iode repositories synchronized.

Note: I am already working to change the Python IODE by using `nanobind` AND making the Python IODE functions to call the C++ API. `nanobind` is much more adapted than Cython to bind a C++ API.

So, try to not develop too much the Cython part in the next few months as the Python will be eventually connected to the C++ API.

If you add new Python functions, please add tests for them. That way, it will be easier for me to translate the Cython function to nanobind function and to test them directly.

Reply:
Ok. I'm in the process of implementing the "Execute Identities" fonctions in C. Maybe (one of) the last python function implemented in Cython.

Reply:
> I'm in the process of implementing the "Execute Identities" fonctions in C. Maybe (one of) the last python function implemented in Cython.

As long as you a test function for it.